test-vectors/tx-out-records/build.rs

```diff
@@ -100,6 +102,7 @@ fn generate_tx_out_record_data() -> TxOutRecordData {
             let e_fog_hint =
                 FogHint::from(&recipient_public_address).encrypt(&fog_pubkey, &mut rng);
             TxOut::new(
+                BlockVersion::TWO,
```

Comment:
Why TWO here and not MAX?

Reply:
Because this is a test vector, so the idea is, we build some tx data, and commit it to the repo as a test vector. Then as a unit test we check that it's hash matches some specific value, so that we can avoid making breaking changes by accidentally changing the hash of old data.